src/controller/BrandController.ts

```diff
@@ -0,0 +1,30 @@
+import { Get, JsonController, Param, QueryParam } from 'routing-controllers';
```

Comment:
This doesn't belong in this MR

Reply:
I created this file here because it was not yet implement on the develop branch wherewas I needed it. I will rebase develop when @nokiiam branch will be merged.